packages/renderer/src/lib/pod/PodCreateFromContainers.svelte

```diff
@@ -291,7 +293,7 @@ function getWarningText(): string {
 
       <div class="w-full grid justify-items-end mt-5">
         <div>
-          <Button type="link" on:click={() => router.goto('/containers')}>Close</Button>
+          <Button type="link" on:click={handleNavigation.bind(undefined, { page: NavigationPage.CONTAINERS })}>Close</Button>
```

Comment:
Not sure to get why bind is required here

Reply:
To avoid 

```ts
on:click={() => handleNavigation({ page: NavigationPage.CONTAINERS })}
```

I find it nicer to have a bind